src/Microsoft.TemplateEngine.Abstractions/Microsoft.TemplateEngine.Abstractions.csproj

```diff
@@ -4,4 +4,10 @@
     <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
     <Description>Contracts for extending Template Engine</Description>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net45'">
```

Comment:
This doesn't look like it's needed

Reply:
IInstallUnitDescriptoryFactory interface needs it because one of the method signatures has JObject